packages/edit-site/src/components/template-details/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default function TemplateDetails( { template } ) {
+	if ( ! template ) {
+		return null;
+	}
+	const templateDescription = __(
+		'In the future, we will display more information about the template here.'
+	);
+
+	return (
+		<div className="edit-site-template-details">
+			<p className="edit-site-template-details__heading">
+				{ __( 'Template details' ) }
+			</p>
+			<p>{ `${ __( 'Name' ) }: ${ template.slug }` }</p>
```

Comment:
We should link this to corresponding item in navigation sidebar. Clicking on it should open the menu and activate the item. This is fine to do in a follow-up though.

Reply:
Ah, I see. That makes sense, I vote to do it in a follow-up :)